extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -9,7 +9,7 @@ import {CanRender} from '#preact/contextprops';
 
 import {ActionInvocation} from '#service/action-impl';
 
-import {waitFor} from '#testing/test-helper';
+import {waitFor} from '#testing/helpers/service-helpers';
```

Comment:
micro-nit: I like context based hierarchies so `#testing/helpers/service` might be a better name.